sourcecode/apis/contentauthor/public/css/ndlah5p-youtube.css

```diff
@@ -0,0 +1,9 @@
+.h5p-dialog-box~.h5p-dialog-box{
```

Comment:
These rules **have** to be narrowed down to the NDLA video browser, otherwise other components risk being affected.